projects/storefrontlib/cms-components/misc/icon/fontawesome-icon.config.ts

```diff
@@ -45,7 +45,9 @@ export const fontawesomeIconConfig: IconConfig = {
       SORT_DOWN: 'fas fa-sort-down',
       SORT: 'fas fa-sort',
       EXPAND_ARROWS: 'fas fa-expand',
+      FILE: 'fas fa-file',
```

Comment:
This is breaking change. This IconConfig can be augmented in "account summary".

Reply:
moved to bottom to avoid breaking change
https://sap.github.io/spartacus-docs/breaking-changes/#avoiding-breaking-changes

Reply:
Not a breaking change in the end